pyinjective/composer.py

```diff
@@ -900,6 +900,24 @@ def MsgVote(
         return cosmos_gov_tx_pb.MsgVote(
             proposal_id=proposal_id, voter=voter, option=option
         )
+    
+    def MsgPrivilegedExecuteContract(self, sender: str, contract: str, msg: str, **kwargs):
+        return injective_exchange_tx_pb.MsgPrivilegedExecuteContract(
+            sender=sender,
+            contract_address=contract,
+            data=msg,
+            funds=kwargs.get('funds') # funds is a string of Coin strings, comma separated, e.g 100000inj,20000000000usdt
+        )
+
+    def MsgInstantiateContract(self, sender, admin: str, code_id: int, label: str, message: bytes, **kwargs):
```

Comment:
Please add the type hint to the sender param and also to the return value of the function